xml/Makefile.am

```diff
@@ -31,7 +31,7 @@ RNGdir			= $(dtddir)
 
 # Sorted list of available numeric RNG versions,
 # extracted from filenames like NAME-MAJOR[.MINOR][.MINOR-MINOR].rng
-RNG_numeric_versions    = $(shell ls -1 *.rng \
+RNG_numeric_versions    = $(shell echo $(wildcard $(top_srcdir)/xml/*.rng) \
```

Comment:
The regression test scripts do not work correctly...
```
Relax-NG parser error : xmlRelaxNGParse: could not load /home/travis/build/ClusterLabs/pacemaker/xml/pacemaker-1.0.rng
```

It should be looking in mkbuild/xml, not the pristine source directory.